dotnet/src/SemanticKernel.Abstractions/Functions/EventHandlerWrapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the main namespace
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Flag abstract class for storing event handler and event args for function events.
+/// </summary>
+public abstract class EventHandlerWrapper
```

Comment:
An empty abstract class is a code smell. We have been working to eliminate empty interfaces. Why is this needed?

At a minimum we should explain this in the class documentation.

Reply:
Agree, will change to code to avoid this.